### DISCUSSION TITLE
MRMS GRIB2 table update

### DISCUSSION THREAD

**Discussion by hertneky** [dtcenter/METplus#1467] [Configuration] [answered]
The Grib2 table for MRMS provided in [MET](https://github.com/dtcenter/MET/blob/main_v10.0/met/data/table_files/grib2_mrms.txt) is quite out of date.

Here is the current table info provided on the [MRMS](https://www.nssl.noaa.gov/projects/mrms/operational/tables.php) site.

What is the ideal way to get this updated and is this something I can help with?

**Comment by JohnHalleyGotway** [comment, marked as answer]
@hertneky thanks for bringing this to our attention. We just created the MET version 10.1.0 beta6 release, which is intended as the last development version for the MET version 10.1.0. We are now in the RC1 (release candidate one) development cycle.

I'll create a GitHub issue to update that table for the RC1 cycle. And I'll list the two of us on that one.

**Reply by hertneky** [reply]
@John Thanks for creating that issue. For now, I have created a custom grib2_mrms.txt, that adds some of the new variables Mike and I are using (e.g. MultiSensor_QPE fields, etc) and was able to run MET on the files with that.
